src/client/tcp.rs

```diff
@@ -26,6 +26,20 @@ pub async fn connect_slave(socket_addr: SocketAddr, slave: Slave) -> Result<Cont
     Ok(context)
 }
 
+/// Connect to a physical, broadcast, or custom Modbus device,
```

Comment:
- Please try to use short sentences
- The initial line or sentence should be the summary, followed by one or more paragraphs with details.

But let's wait until we decided if you use an implicit constant for the time being before we are switching to a configurable parameter when introducing the builder pattern.